cctk/orca_file.py

```diff
@@ -17,41 +17,29 @@ class OrcaJobType(Enum):
     All jobs have type ``SP`` by default.
     """
 
-    SP = "sp"
+    SP = ("sp", ["energy", "scf_iterations"])
```

Comment:
Could you also extract the timings?  Or you must be doing it already for the benchmarks?